### PR TITLE
Add an article

### PR DESCRIPTION
**Pull request by tn-tran**
## Checklist

* [ ] The links are in the correct format: ``[Title of the Article](link), by [@author/`s Twitter username](link-for-twitter)``
* [ ] I added my self to the credits with my GitHub account: ``[GitHub account](link-to-github]``

## Optional

* [ ] Article is not more than 2 weeks old
* [ ] Article wasn't submitted before

### CODE REVIEW

**Comment by rbarbosa** [comment]
Hi @tn-tran!
You don't (and you shouldn't) have to change the `PULL_REQUEST_TEMPLATE.md`. When creating the PR the template is used.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ## Checklist
 
-* [ ] The links are in the correct format: ``[Title of the Article](link), by [@author/`s Twitter username](link-for-twitter)``
-* [ ] I added my self to the credits with my GitHub account: ``[GitHub account](link-to-github]``
+* [x] The links are in the correct format: ``[Title of the Article](link), by [@author/`s Twitter username](link-for-twitter)``
+* [x] I added my self to the credits with my GitHub account: ``[GitHub account](link-to-github]``
```
Comment (FranciscoAmado) [inline .github/PULL_REQUEST_TEMPLATE.md]: Could you take the opportunity and fix the bracket in `(link-to-github]` to `(link-to-github)`?.. 
it drives me crazy 😅

Comment (rbarbosa) [reply .github/PULL_REQUEST_TEMPLATE.md]: I will fix that... 😂

**Comment by rbarbosa** [comment]
Going to close this...

**Pull request closed without merging**